service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,7 +80,11 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
         Document document = uploadDocumentService.uploadPDF(userId, authorization, pdf, buildFileName(caseDetails));
 
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
```

Comment:
formatter could be renamed to sdf, I have seen it this way in a few places. Thoughts?

Reply:
👍